web_external/js/views/widgets/WmsLayersListWidget.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+/**
+* This widget displays the list of WMS layers
+*/
+minerva.views.WmsLayersListWidget = minerva.View.extend({
+
+    events: {
+        'submit #m-add-layers-form': function (e) {
+            e.preventDefault();
+            var wmsSource = this.dataset;
+            var hostName = (wmsSource.get('meta').minerva.wms_params.base_url).slice(0, 28);
+            var that = this;
+            $('input[type=checkbox]').each(function () {
```

Comment:
change this to `_.bind` for consistency (at least I hope we are being consistent).  That is, remove `var that = this;`, and

    $('input[type=checkbox]').each(_.bind(function () {
        // some stuff
    }, this));

 and then you can change the `that`s to `this`.